tools/accessed-funds-calculator/src/tokens.rs

```diff
@@ -0,0 +1,333 @@
+#[derive(Debug)]
+pub enum TokenName {
+    Usdc,
+    Cqt,
+    Usdt,
+    Frax,
+    Wbtc,
+    Iag,
+    Weth,
+    Dai,
+    C3,
+    Fxs,
+    Cards,
+    Hbot,
+    Sdl,
+    Gero,
+}
+
+#[derive(Debug)]
+pub struct Token {
+    pub name: TokenName,
+    pub id: String,
+    pub decimals: f64,
+    pub contract_address: String,
+    pub recovered_total: f64,
+}
+
+impl Token {
+    pub fn get_instance_of(token_name: TokenName) -> Token {
+        match token_name {
+            TokenName::Usdc => Token {
```

Comment:
if all of these are effectively constants, they should be named statics

you could also re-use assets.rs from the nomad nft server

```pub struct Token {
    pub name: TokenName,
    pub id: &'static str,
    pub decimals: f64,
    pub contract_address: &'static str,
    pub recovered_total: f64,
}

static CQT: Token = oken {
                name: TokenName::Cqt,
                id: "covalent",
                decimals: 18.0,
                contract_address: "0xD417144312DbF50465b1C641d016962017Ef6240",
                recovered_total: 34_082_775.751_599_7,
            };
```